### DISCUSSION TITLE
Does sbt consider artifact repos declared in a dependency's POM to resolve transitive dependencies?

### DISCUSSION THREAD

**Discussion by sschuberth** [sbt/sbt#7156] [Q&A]
I'm looking for similar information like Maven [provides](https://maven.apache.org/guides/mini/guide-multiple-repositories.html#repository-order), i.e. whether any artifact repositories that are declared as part of POMs of a dependency are taken into account by sbt to resolve the dependency's transitive dependencies. While Maven does use repositories from "effective POMs from dependency path to the artifact", [Gradle ignores these](https://docs.gradle.org/current/userguide/declaring_repositories.html#strict_limitation_to_declared_repositories). How does sbt behave here?

**Comment by eed3si9n** [comment]
This might be related to:
- https://github.com/sbt/sbt/issues/4531
- https://github.com/coursier/coursier/issues/1390

In general, sbt outsources the dependency management to Couriser (and previously Apache Ivy), so any fundamental discussions around POM probably should be brought up to coursier/coursier.
